war.yml

```diff
@@ -6,6 +6,7 @@ application:
       artifact-url: bind(checker#connection.url)
   configuration:
     configuration.war-path: "application.war"
+    configuration.docker-image-id: "qubell/artifact-builder:latest"
```

Comment:
The same concern as in https://github.com/qubell-bazaar/component-petclinic-docker/pull/7#discussion_r49423494 applies here